example.rules

```diff
@@ -0,0 +1,9 @@
+# The count of CPUs per node, useful for getting CPU time as a percent of total.
+instance:node_cpu:count = count(node_cpu{mode="idle"}) without (cpu,mode)
```

Comment:
No, look again, this filters by one mode, so it returns the correct count.  I've tested this to make sure.